packages/chart/examples/testposorig.json

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
@jayb what is this file being used for?

Reply:
I would write a ticket in Jira with a description of the bug that you found. You can attach this config in the actual Jira ticket as part of the steps to reproduce the issue. We don't need to save this in our repository.

Reply:
Once you have the ticket number add the ticket number to the PR. eg: `fix: dev-1234 first grid line`

Reply:
I created DEV-9142, moved the test file there and updated the PR title per your recommendation.